go/kbfs/libkbfs/favorites.go

```diff
@@ -339,6 +342,7 @@ func (f *Favorites) handleReq(req *favReq) (err error) {
 	// quickly when offline.
 	needFetch := (req.refresh || f.favCache == nil) && !req.clear
 	wantFetch := f.config.Clock().Now().After(f.cacheExpireTime) && !req.clear
+	changed := false
```

Comment:
Heh just realized I made the almost exact change in my PR: https://github.com/keybase/client/pull/17530/files#diff-5919763ea12fd1848e70a4ca1d6ce411R446

I'll incorporate yours after yours is merged